### DISCUSSION TITLE
Did not able to access the event, save icon in Execute option.

### DISCUSSION THREAD

**Discussion by abmuslim** [InterDigitalInc/AdvantEDGE#367] [Q&A] [answered]
Hello,
I deploy my scenario in the Execute option and the only option I am able to access in Terminate, I did Terminate the App instances but when I deploy again I only see the Terminate option. I am new to AdvantEdge and I am using its GUI.
Regards,

![image](https://user-images.githubusercontent.com/55253316/178751729-3818ffbb-c6ff-4b4c-b5af-5256a9646c8f.png)

**Comment by dilallkx** [comment, marked as answer]
Hi @abmuslim,

Thank you for reaching out!

The issue you observe is typically due to a failure during scenario deployment. If you scroll down on the Execute tab you should see a list of network elements being deployed from your scenario. The save, event & dashboard buttons become enabled only once all network elements are in "Running" state. If one element is stuck in another state (e.g. "Pending") you will need to determine why your k8s pod is failing to start (e.g. bad image name, etc.). You can check the k8s pods logs to debug this.

I also recommend trying to deploy the demo1 scenario to get a better understanding and more familiarity with the platform. The documentation on platform usage and demo1 can be found here:
https://interdigitalinc.github.io/AdvantEDGE/docs/usage

I hope this helps.
Regards.

**Reply by abmuslim** [reply]
Thank you for your reply. I get it now.
How can I implement Demo 1 on GUI because it is for Terminal right? If no then how I configure my k8 pods in GUI nodes? I mean which container images I should put and stuff like that?

**Reply by dilallkx** [reply]
The demo1 scenario is available in the repo, and the procedure for importing and deploying demo1 is documented here:
https://interdigitalinc.github.io/AdvantEDGE/docs/usage/usage-basic/
